float64[6] softness
---